packages/property-grid/src/components/PropertyGridUiItemsProvider.tsx

```diff
@@ -0,0 +1,46 @@
+import { I18N } from "@bentley/imodeljs-i18n";
+import {
+  AbstractWidgetProps,
+  StagePanelLocation,
+  StagePanelSection,
+  UiItemsProvider,
+} from "@bentley/ui-abstract";
+
+import { PropertyGrid, PropertyGridProps } from "./PropertyGrid";
+import React = require("react");
+import { connect } from "react-redux";
+import { FrameworkState } from "@bentley/ui-framework";
+
+// Map framework state to property grid props
+const mapStateToProps = (state: FrameworkState): PropertyGridProps => {
```

Comment:
anyway to make sure that the consumer has setup the UiFramework redux store? Should probably have default values or throw an error if not

Reply:
Added check but not in the mapStateToProps. The check is in the PropertyGridManager.ts initialize function. StateManager class can be checked to ensure UiFramework store is initialized properly.